recipes/noiseflow/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set org = "shakeflow" %}
+{% set name = "noiseflow" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 20736acea7dde31bbd3c6c6c3fbe922bedf242dd0b64bcd02a4557b4d566b427
+
+build:
+  number: 0
+  script: 
+    - git clone https://github.com/xtensor-stack/xtensor-fftw extern/xtensor-fftw  # [unix]
+    - git clone https://github.com/kfrlib/kfr extern/kfr  # [unix]
+    - export LLVM_OPENMP="1"  # [osx]
+    - export NOISEFLOW_USE_CPP="1"  # [linux]
```

Comment:
I don't know about the package, so just to check: you don't want this on other platforms

Reply:
Yes, I don't want to use cpp_module in Win platform.

Reply:
And not on Mac either?

Reply:
Ohhhh, I see. It should be `- export NOISEFLOW_USE_CPP="1"  # [unix]`